code/__DEFINES/sound.dm

```diff
@@ -8,11 +8,12 @@
 #define CHANNEL_AMBIENT_EFFECTS 1018
 #define CHANNEL_AMBIENT_MUSIC 1017
 #define CHANNEL_AMBIENT_BUZZ 1016
+#define CHANNEL_REACTOR_ALERT 1015 //austation -- reactor sounds
 
 //THIS SHOULD ALWAYS BE THE LOWEST ONE!
 //KEEP IT UPDATED
 
-#define CHANNEL_HIGHEST_AVAILABLE 1015
+#define CHANNEL_HIGHEST_AVAILABLE 1014 //austation -- reactor sounds
```

Comment:
what?
is this part of modularized code?
I don't see how this would not break things on its own.

Reply:
You always have to change it, check the comments above